README.md

```diff
@@ -2,4 +2,15 @@ The filter order dependency resolver accepts dependency declarations in
 the form of injected beans and reorders servlet filters in order to
 accord to the desired dependencies.
 
-See the classes' Javadocs for usage.
+Here is an sample use.  In it, B is said to depend on A, so in the final
```

Comment:
`a sample`

Reply:
Oops, thanks.  Fixed in 109027bd7282ab11b577be81853556080351b8fc.

Reply:
Further improved based on in-person chatting with @fsosa in 8cd00a835574d91338003878780f6c82bef35ea6.